scripts/zkai_d128_two_slice_accumulator_backend_gate.py

```
@@ -0,0 +1,844 @@
+    ):
+        raise D128TwoSliceAccumulatorBackendError("draft payload must not include mutation metadata")
+    expect_equal(payload.get("summary"), expected_summary, "summary")
+
+
+def _validate_case_metadata(payload: dict[str, Any]) -> tuple[int, int]:
+    has_cases = "cases" in payload
+    has_case_count = "case_count" in payload
+    has_all_mutations_rejected = "all_mutations_rejected" in payload
+    has_mutation_inventory = "mutation_inventory" in payload
+    if not (has_cases or has_case_count or has_all_mutations_rejected or has_mutation_inventory):
+        raise D128TwoSliceAccumulatorBackendError("mutation metadata missing")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)